definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -4707,7 +4707,7 @@
     "LiveFrom": "25/06/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorUpdateDN",
-    "UserRole": "[PETSOLICITOR]",
+    "UserRole": "caseworker-divorce-solicitor",
```

Comment:
Reverting the UserRole changes introduced previously, until further info about solicitor journey